Examples/Algorithms/Geant4/src/Geant4Manager.cpp

```diff
@@ -80,6 +81,12 @@ std::shared_ptr<Geant4Handle> Geant4Manager::currentHandle() const {
 
 std::shared_ptr<Geant4Handle> Geant4Manager::createHandle(
     const std::string& physicsList) {
+  if (physicsList == "MaterialPhysicsList") {
+    return createHandle(
+        std::make_unique<MaterialPhysicsList>(
+            Acts::getDefaultLogger("MaterialPhysicsList", Acts::Logging::INFO)),
+        physicsList);
+  }
```

Comment:
can this be done below with `registerPhysicsListFactory`?

Reply:
Done.